pkg/processor/trigger/kafka/types.go

```diff
@@ -170,6 +171,9 @@ func NewConfiguration(id string,
 
 		// allow changing explicit ack mode via annotation
 		{Key: "nuclio.io/kafka-explicit-ack-mode", ValueString: &explicitAckModeValue},
+
+		// allow changing explicit ack mode via annotation
+		{Key: "nuclio.io/wait-explicit-ack-during-rebalance-timeout", ValueString: &triggerConfiguration.WaitExplicitAckDuringRebalanceTimeout},
```

Comment:
Add a test to verify what takes precedence when both the annotation and trigger configuration are set.

Reply:
@TomerShor will add 👍  I guess that annotation will be prioritised.

Reply:
Added test, annotations value has a priority